README.md

```diff
@@ -1,16 +1,29 @@
-# Charm for MongoDB on Kubernetes
+# Charmed MongoDB on Kubernetes
+## Overview
 
-## Description
+The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database. It is an open source, end-to-end, production ready data platform on top of cloud native technologies.
 
-Fully supported solution for production-grade MongoDB on Kubernetes from Canonical.
+MongoDB is a popular NoSQL database application. It stores its data with JSON-like documents creating a flexible experience for users; with easy to use data aggregation for data analytics. It is a distributed database, so vertical and horizontal scaling come naturally.
 
-Run the developer's favourite document database - MongoDB! Charm for MongoDB is a
-fully supported, automated solution from Canonical for running production-grade
-MongoDB on Kubernetes. It offers simple, secure and highly available setup with
-automatic recovery on failover. The solution includes scaling and other capabilities.
+This operator charm deploys and operates MongoDB on Kubernets. It offers features such as replication, TLS, password rotation, and easy to use integration with applications. The Charmed MongoDB Operator meets the need of deploying MongoDB in a structured and consistent manner while allowing the user flexibility in configuration. It simplifies deployment, scaling, configuration and management of MongoDB in production at scale in a reliable way.
+
+## Requirements 
+- 2GB of RAM.
```

Comment:
Would it be worth adding `at least` before "2GB of RAM" and "2 CPU.."? or is that implied already?